lua/hop/hint.lua

```diff
@@ -15,11 +19,14 @@ end
 -- Regex hint mode.
 --
 -- Used to hint result of a search.
-function M.by_searching(pat, plain_search)
+function M.by_searching(pat, plain_search, direction, same_line, opts)
   if plain_search then
     pat = vim.fn.escape(pat, '\\/.$^~[]')
   end
   return {
+    direction = direction,
+    linewise = false,
```

Comment:
I checked the rest of the PR and I don’t quite get what `linewise` is used for vs. `same_line` / `one_shot`.